### ISSUE TITLE
Neural Network only outputting 1 regardless of the input

### ISSUE THREAD

**Issue by spudzee** [fchollet/keras#500]
Here's my code: http://pastebin.com/px6e2nVv

I want the network to, when given a sequence of letters/digits, predict the next letter/digit. I want it to eventually produce meaningful language/grammar.

At the moment it produces this: http://pastebin.com/PwDamLS2 (I cut out epochs 101-900 because they're basically the same thing just a different epoch number). As you can see the result is just 1's. I looked at the original outputs before I round/int them. They are all the value "1.0", something strange is happening.

I tried looking at the input (before and after encoding), and it appears everything is working properly there. The input arrays aren't repeating, and the individual input values all have their respective numerical values.

**Comment by fchollet** [comment]
Softmax on an array of size 1 is always 1, by definition of softmax.

**Comment by spudzee** [comment]
Ok I made a few modifications to the code but it's still producing strange results. My code: http://pastebin.com/sg9fdCZM

the outputs aren't strictly 1's anymore, it's actual floats with understandable values. But the problem I'm having now is the output is staying the same regardless of the input. Basically it's just outputing 100 commas.

**Comment by dattanchu** [comment]
I am running into a similar problem with my simple dense neural network of Input(5) -> Dense (512) -> Dense (512) -> Dense (1) outputting the same float value regardless of input. 

@spudzee do you have any insight?
